clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -216,6 +217,14 @@ public synchronized boolean updateRequested() {
         }
     }
 
+    public synchronized Uuid topicId(String topicName) {
```

Comment:
Can you document that this returns null if the topicId does not exist or is not known? Similarly for `topicName`.

Reply:
Will do